dotCMS/src/main/java/com/dotmarketing/portlets/workflows/model/MultiUserReferenceParameter.java

```diff
@@ -0,0 +1,113 @@
+package com.dotmarketing.portlets.workflows.model;
+
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.Role;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UtilMethods;
+import com.liferay.util.Validator;
+import java.util.StringTokenizer;
+
+/**
+ * This type of workflow actionlet parameter allows content managers to enter a comma-separated list
+ * of values used to specify different system users. This class provides 3 mechanisms to specify
+ * users in you actionlet:
+ * <ol>
+ * <li>The user ID.</li>
+ * <li>The user's email address.</li>
+ * <li>The role key (this includes all the users associated to the specified role).</li>
+ * </ol>
+ *
+ * @author Jose Castro
+ * @version 4.3.0
+ * @since Jan 9, 2018
+ */
+public class MultiUserReferenceParameter extends WorkflowActionletParameter {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a "static final long serialVersionUID" field to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2057)